release-team/role-handbooks/bug-triage/README.md

```diff
@@ -1,195 +1,223 @@
 Bug Triage Release Team Role
 ==============================
 
-Your job is to make sure that bugs which affect the release are dealt with in a timely fashion.  Bugs can appear either as GitHub Issues or Pull Requests (PRs).  
+Primarily, your job is to make sure that bugs which affect the release are dealt with in a timely fashion.  
+Bugs can appear either as GitHub Issues or Pull Requests (PRs).  
 
-Primarily, you will be polling bugs, checking status, sending reminders and questions to contributors and SIG leads, and publishing summary reports.  The role has been interchangeably called "Bug Triage" and "Issue Triage", but the general term "issue" can be conflated with the specific GitHub usage of `is:issue` artifact type.  It is important to consider both `is:issue` and `is:pr` GitHub artifacts when triaging bugs.
+A list of common tasks is:
+- Polling for bugs (mainly issues with the label `kind/bug`)
+- Checking status of open issues / PRs
+- Sending reminders to relevant assignees / owners / sig-leads
```

Comment:
This is passive language which worries me relative to an experience in 1.12.  Bug triage needs to be active.  In order to have information to inform the next bullet ("Publishing summary reports"), the bug triage role must involve more than just sending reminders to the right folks and bias toward action...interlocking with those folks to get them to share status, update issues/PRs, clarify a situation, and enable next level decision making.  If one only sends reminders, there is a high likelihood of no additional information being collected, because reminders and notifications are lost in a sea of reminders and notifications.

Reply:
indeed. this initial list is meant to be a short subset of the tasks where I've written about the others later on, but good to include more details at the start

Reply:
changed it to 'some common tasks include:' and added some more tasks